Drivers/Src/Contactors.c

```diff
@@ -1,66 +1,72 @@
 /* Copyright (c) 2020 UT Longhorn Racing Solar */
 
-/**
- * Source file to mimic GPIO port communication
- */ 
 
 #include "Contactors.h"
 
+
+#define CONTACTORS_PORT     PORTC
+#define ARRAY_PRECHARGE_PIN 10
+#define ARRAY_CONTACTOR_PIN 11
+#define MOTOR_CONTACTOR_PIN 12
```

Comment:
Why not use the `GPIO_Pin_x` macros instead of `10`, `11`, and `12`, so you don't have to deal with bit shifting?